cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -8842,6 +8857,26 @@ ir.cpp:
 # 1689|     v1689_12(void)                             = AliasedUse                   : ~m?
 # 1689|     v1689_13(void)                             = ExitFunction                 : 
 
+# 1693| int global_1
+
+# 1695| int global_2
+# 1695|   Block 0
+# 1695|     v1695_1(void)       = EnterFunction   : 
+# 1695|     r1695_2(glval<int>) = VariableAddress : 
```

Comment:
This `VariableAddress` is missing the variable. Could this be the cause of some of those consistency failures? Did you remember to create an `IRVariable` for the global being initialized?

Reply:
I just added an `IRVariable`, which fixes the missing variable here but doesn't solve the consistency failures.